core/src/Dscp/Core/Fees.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE StrictData #-}
+
+-- | Trnsaction fees processing and calculation.
+module Dscp.Core.Fees
+       ( Fees (..)
+       , FeeCoefficients (..)
+       , calcFeeTx
+       , calcFeePub
+       , calcFeeG
+       , noFees
+       , fixFees
+       ) where
+
+import Codec.Serialise (Serialise (..))
+
+import Dscp.Core.Foundation
+import Dscp.Crypto
+import Dscp.Util
+
+-- | Amount of transaction fees.
+newtype Fees = Fees { unFees :: Coin }
+    deriving (Eq, Ord, Show, Generic, Hashable, Bounded)
+
+instance Serialise Fees
+
+-- | Setup for fee calculation.
+data FeeCoefficients = FeeCoefficients
+    { fcMinimal       :: Coin  -- ^ How much is the base fee?
```

Comment:
I believe that minimal fee for publications should be different from minimal fee for money transactions.